server/app/lib/message.rb

```diff
@@ -0,0 +1,33 @@
+class Message
```

Comment:
I like this idea to put the messages in one place - apart from always knowing where to make a string change, it will be easier to at a later point transfer the strings somewhere else so that they can be translated, for example